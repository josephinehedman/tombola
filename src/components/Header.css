.header {
  background-color: #d59203;
  height: 100px;
  margin-bottom: 2rem;
}

a {
  color: white;
  text-decoration: none;
}

.header__navbar {
  height: 100%;
  display: flex;
  align-items: center;
  justify-content: space-around;
}

.nav-list {
  display: flex;
  justify-content: space-evenly;
}

.nav-list__links:hover {
  text-decoration: underline;
}

.nav-list__links {
  list-style-type: none;
}

.nav-list__links--right {
  margin-left: 2rem;
}

@media all and (min-width: 840px) {
  .header {
    font-size: 1.5rem;
    margin-bottom: 3rem;
  }
}
