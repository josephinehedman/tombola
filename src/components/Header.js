import { Link } from 'react-router-dom';
import './Header.css';

const Header = () => (
  <header className="header">
    <nav className="header__navbar">
      <Link to="/tombola">
        <h1 className="navbar__logo">Tombola</h1>
      </Link>
      <ul className="nav-list">
        <Link to="/about">
          <li className="nav-list__links nav-list__links--left">
            About
          </li>
        </Link>
        <Link to="/activity">
          <li className="nav-list__links nav-list__links--right">
            Activity
          </li>
        </Link>
      </ul>
    </nav>
  </header>
);

export default Header;
