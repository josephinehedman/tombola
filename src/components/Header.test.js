import Header from './Header';
import renderer from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';

describe('Header component', () => {
    test('renders header that matches snapshot', () => {
        const header = renderer
        .create(<BrowserRouter><Header/></BrowserRouter>)
        .toJSON();
       expect(header).toMatchSnapshot();
    });
});