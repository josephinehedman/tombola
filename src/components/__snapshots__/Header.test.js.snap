// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component renders header that matches snapshot 1`] = `
<header
  className="header"
>
  <nav
    className="header__navbar"
  >
    <a
      href="/"
      onClick={[Function]}
    >
      <h1
        className="navbar__logo"
      >
        Tombola
      </h1>
    </a>
    <ul
      className="nav-list"
    >
      <a
        href="/about"
        onClick={[Function]}
      >
        <li
          className="nav-list__links nav-list__links--left"
        >
          About
        </li>
      </a>
      <a
        href="/activity"
        onClick={[Function]}
      >
        <li
          className="nav-list__links nav-list__links--right"
        >
          Activity
        </li>
      </a>
    </ul>
  </nav>
</header>
`;
